src/styles/blocks/card.scss

```diff
@@ -0,0 +1,48 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  max-width: $card-width;
+
+  padding: 32px 16px 16px;
+
+  background: $bgc-light;
+  border: 1px solid $bgc-grey;
+  border-radius: 5px;
+
+  &__img {
+    padding: 0 3px;
+
+    width: 166px;
+    height: 134px;
+  }
+
+  &__article-title {
+    margin-top: 40px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
```

Comment:
it's a bad practice to line height in percents, better to set is a `line-height: 1.4;`